application/views/todo_modal.php

```diff
@@ -0,0 +1,18 @@
+      <!-- //header -->
+      <div class="modal-header">
+            <h5 class="modal-title" id="staticBackdropLabel">ADD ITEM TO TODOLIST</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <!-- body -->
+      <div class="modal-body">
```

Comment:
where is the form tag?

Reply:
im using ajax to push data into database

Reply:
I see it now. Ok2x.